src/PhpSpec/Console/ContainerAssembler.php

```diff
@@ -301,7 +308,8 @@ private function setupGenerators(IndexedServiceContainer $container)
                 $c->get('console.io'),
                 $c->get('code_generator.templates'),
                 $c->get('util.filesystem'),
-                $c->get('code_generator.writers.tokenized')
+                $c->get('code_generator.writers.tokenized'),
+                new CodeGenerator\Generator\Argument\StringBuilder()
```

Comment:
define this as a service so we can reuse it elsewhere (and so extensions can decorate it)